docs/legal/TermsOfService.rst

```diff
@@ -0,0 +1,122 @@
+======================
+Terms and Conditions
```

Comment:
assuming this has been reviewed by a legal expert of some sort?

Reply:
Khirul vai is reviewing it, his review is still going on.